test/index.spec.ts

```diff
@@ -22,6 +23,29 @@ describe('identify', () => {
     ]);
   });
 
+  it('should identify custom parameters', () => {
```

Comment:
Can we have a test to validate that setting `paramTypes` will override the default parameter matching for a given adapter, e.g. if I have `positional: true` set, then numbered params will be ignored, or whatever the behavior is with `sql-formatter` in dealing with this.